stairctl/src/main/java/bio/terra/stairctl/commands/Output.java

```diff
@@ -10,13 +10,13 @@
 import java.util.Comparator;
 import java.util.List;
 import java.util.Optional;
-import javax.annotation.Nullable;
 import org.apache.commons.lang3.ClassUtils;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.tuple.ImmutablePair;
 import org.apache.commons.lang3.tuple.Pair;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.lang.Nullable;
```

Comment:
`javax.annotation.Nullable` was coming from spotbugs (https://www.javadoc.io/doc/com.google.code.findbugs/jsr305/latest/javax/annotation/Nullable.html).

There seem to be lots of options for @Nullable, so I went with a package that was already available with our current dependencies.

Reply:
I think a better fix would be to use `jakarta.annotation.Nullable`, as it doesn't require a framework dependency.

Reply:
It didn't seem that was available with our current dependencies/framework, but perhaps IntelliJ was lying to me?

Reply:
Yes, it might need an additional dependency like https://mvnrepository.com/artifact/jakarta.annotation/jakarta.annotation-api

Reply:
Right, I started looking into adding that, but then punted since we use `org.springframework.lang.Nullable` in some of our other repos.